src/tito/release/distgit.py

```diff
@@ -49,6 +49,10 @@ def __init__(self, name=None, tag=None, build_dir=None,
         self.git_branches = \
             self.releaser_config.get(self.target, "branches").split(" ")
 
+        # check .tito/releasers.conf
+        if self.releaser_config.has_option(self.target, "remote_git_name"):
+            overwrite_checkout = self.releaser_config.get(self.target, "remote_git_name")
+        # fallback to old, incorrect, location: .tito/packages/tito.props
         if self.config.has_option(self.target, "remote_git_name"):
```

Comment:
Should just be an elif here so the correct location wins out.

Otherwise LGTM!